src/module/mod.rs

```diff
@@ -352,13 +351,22 @@ impl Module {
                 section.apply_code_transform(&cx.code_transform);
             }
 
-            cx.custom_section(&section.name())
-                .encoder
-                .raw(&section.data(&indices));
+            cx.wasm_module.section(&wasm_encoder::CustomSection {
+                name: section.name().into(),
+                data: section.data(&indices).into(),
+            });
         }
 
+        let out = cx.wasm_module.finish();
         log::debug!("emission finished");
-        wasm
+
+        // let mut validator = Validator::new();
+        // if let Err(err) = validator.validate_all(&out) {
+        //     eprintln!("{:?}", err);
+        //     panic!("Unable to validate serialized output");
+        // }
+
```

Comment:
Any reason to keep this commented-out code?
```suggestion
```

Reply:
It's quite useful for testing, but it could be removed.